crates/trippy-tui/src/frontend/columns.rs

```diff
@@ -426,6 +430,7 @@ mod tests {
     #[test_case(ColumnType::StdDev, "StDev")]
     #[test_case(ColumnType::Status, "Sts")]
     fn test_column_display_formatting(c: ColumnType, heading: &'static str) {
+        let _ = crate::locale::init(Some("en"));
```

Comment:
This (and the one below) is due to `i18n_embed::select()` being called in `crate::locale::init()`, which is to take `TrippyConfig::tui_locale` into account. Being called on each test invocation shouldn't matter much, it's called only once in the main program. Still, the aesthetics might not be favorable.

Reply:
https://github.com/fujiapple852/trippy/blob/98f3f8e24e4c1e83050dba1a45201d5ade85bf43/crates/trippy-tui/src/lib.rs#L23-L24

Nevertheless, it has to use a global lazy value, we can too, but I decided to go with `locale::init()` to reduce that a bit. (early optimization? heh)